bootup.sh

```diff
@@ -1,19 +1,19 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
nur aus Interesse: ist das einfach aus "best practice"-gründen oder brauchen wir hier die angabe mit env?

Reply:
Ah.... dafür brauchen wir es nä? : 
==> "Soll das Projekt aus den lokalen Dateien gestartet werden, muss eine Datei mit dem Namen .env mit folgendem Inhalt angelegt werden:"

Reply:
Nein, das ist einfach nur BestPraktice.
Die .env-Datei wird von *docker-compose* beim start gelesen und in dem dem Punkt `command` eingefügt.